### PR TITLE
Fix: Adding gray bg and making header bold for group link on gnav

### PR DESCRIPTION
**Pull request by bandana147**
- Adds a gray background option for link header
- Makes gradient link group header bold 

Resolves: [MWPW-146166](https://jira.corp.adobe.com/browse/MWPW-146166)

**Test URLs:**
- Before: https://main--milo--adobecom.hlx.live/drafts/blaishram/index-loggedout-ace0687v4?martech=off
- After: https://mwpw-146166--milo--bandana147.hlx.live/drafts/blaishram/index-loggedout-ace0687v4?martech=off

### CODE REVIEW

File: libs/blocks/global-navigation/base.css [hunk 1]
```diff
@@ -222,6 +222,7 @@
   .feds-navLink[class *= '-gradient'] {
     display: flex;
     border-radius: 4px;
+    font-weight: 600;
```
Comment (mokimo) [inline libs/blocks/global-navigation/base.css:225]: Is it **always bolded**, or just in the gray gradient?

**Review by mokimo — APPROVED** [review]

File: libs/blocks/global-navigation/base.css (hunk 1 above)
Comment (bandana147) [reply libs/blocks/global-navigation/base.css:225]: I was informed that all gradients should be bolded.

Comment (overmyheadandbody) [reply libs/blocks/global-navigation/base.css:225]: @mokimo - for context, we shouldn't be currently using any colorful gradient on production currently, it's still in a test phase. So I think it's safe to apply the rule to all such elements 

And come to think of it, could we confirm with the stakeholder whether they still see a need for all the other gradient classes? Maybe we could remove them already if that test failed.

**Review by overmyheadandbody — APPROVED** [review]
I'd suggest we discuss the need for all the other gradients with the stakeholder, it might be that we can remove them to keep the file clean.

**Comment by bandana147** [comment]
The failing Lighthouse score could be attributed to the usage of a global navigation with multiple fragments including some from other sites, some of which result in 404 errors. Since this change involves only a CSS modification with background color and font weight adjustments, it should have no impact on performance.

**Review by robert-bogos — APPROVED** [review]

**Review by salonijain3 — APPROVED** [review]

**Comment by salonijain3** [comment]
This is a high priority PR as this is needed for ACE0687 which needs be live for 5/7 and 28 days is needed for the process.

**Pull request merged**